connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -223,6 +231,15 @@ ForwardingAdmin forwardingAdmin(Map<String, Object> config) {
         }
     }
 
+    void addClientId(Map<String, Object> props, String role) {
+        String clientId = entityLabel() + (role == null ? "" : "|" + role);
+        props.put(CommonClientConfigs.CLIENT_ID_CONFIG, clientId);
```

Comment:
good point, will do - one note though: since current the control over the client.ids of different nested clients is very limited, I'm not sure if it is actually useful, e.g. in MM2 dedicated mode, the most specific config users can use is something like source.consumer.client.id, but that doesn't differentiate between MirrorSourceTask replication consumers or MirrorCheckpointTask offset sync consumers